modules/redpanda/mounts/bootstrap.yaml.tpl

```diff
@@ -22,3 +22,8 @@ data_transforms_enabled: true
 {{- if .AutoCreateTopics }}
 auto_create_topics_enabled: true
 {{- end }}
+
+{{- range $key, $value := .ExtraBootstrapConfig }}
```

Comment:
Question: how will redpanda behave if users pass incorrect config key-value pairs here?

This is not related to the code itself but to the experience using the module

Reply:
Sorry, missed this. In general, redpanda will simply reject any unrecognized config (the key) and halt. I'm fairly certain the same applies to ill-formed values. For example, a value outside the range of a bounded config. Broker logs should reflect this in detail.

So the experience configuring the module this way should be quite similar to editing a Redpanda cluster config file before firing up Redpanda manually.

Reply:
Thanks for the explanation. Once the CI passes, I'll merge this PR